app/javascript/.eslintrc.js

```diff
@@ -38,6 +40,7 @@ module.exports = {
   },
   plugins: ['import', 'react', 'jsx-a11y'],
   rules: {
+    'import/order': ['error'],
```

Comment:
Enforces import order but is fixable via `eslint --fix` so there is no friction on a contributor.

Reply:
Not a fan? This isn't a deal breaker for me. I'm curious, if it gets fixed automatically on commit, what's the 👎 for? Too rigid a rule? Also, it could be made a warning.

Reply:
I have no idea where that thumbs-down came from 🤷‍♂️ There may just be too much blood in my coffee stream.